tools/modules/EnsEMBL/Web/Component/Tools/TicketDetails.pm

```diff
@@ -160,19 +162,37 @@ sub get_job_summary {
       else {
         my $details   = $exception->{'message'} ? "Error with message: $exception->{'message'}\n" : "Error:\n";
            $details   = $exception->{'stack'} ? $details : $details.$exception->{'exception'} || 'No details';
+        
+        my $helpdesk_link = $hub->url({'type' => 'Help', 'action' => 'Contact', 'subject' => sprintf('Exception in %s Web Tools', $hub->species_defs->ENSEMBL_SITETYPE), 'message' => sprintf("\n\n\n%s with message (%s) (for %s): %s", $exception->{'class'} || 'Exception', $display_message, $url_param, $details)});
 
-        my $helpdesk_details = sprintf 'If the error persists, please contact our <a href="%s" class="modal_link">helpdesk</a> to report this problem.',
-          $hub->url({'type' => 'Help', 'action' => 'Contact', 'subject' => sprintf('Exception in %s Web Tools', $hub->species_defs->ENSEMBL_SITETYPE), 'message' => sprintf("\n\n\n%s with message (%s) (for %s): %s", $exception->{'class'} || 'Exception', $display_message, $url_param, $details)})
-        ;
-
-        $error_div->append_children({
-          'node_name'   => 'div',
+         my $error_details_div = $self->dom->create_element('div', {
```

Comment:
sorry @msmanoj i dont quite understand this change. something doesnt look right here to me. probably best to discuss this over a video call (friday?)

Reply:
Hi @ens-ma7 - I have made the changes as discussed.